docs/source/workflow_inputs.rst

```diff
@@ -3260,9 +3260,9 @@ Each conventional storage water heater is entered as a ``/HPXML/Building/Buildin
          See :ref:`hpxmllocations` for descriptions.
   .. [#] If Location not provided, defaults to the first present space type:
          
-         \- **IECC zones 1-3, excluding 3A**: "garage", "conditioned space"
+         \- **IECC zones 1-3**: "garage", "conditioned space"
          
-         \- **IECC zones 3A, 4-8, unknown**: "basement - conditioned", "basement - unconditioned", "conditioned space"
+         \- **IECC zones 3-8, unknown**: "basement - unconditioned", "basement - conditioned", "conditioned space"
```

Comment:
Good change. This logic is (for better or worse) repeated for each water heater type, can you make the same change to all of them?

Reply:
Just fixed in the latest commit. Thanks for pointing it out!